src/app/content/search/selectors.ts

```diff
@@ -10,3 +11,19 @@ export const query = createSelector(
   localState,
   (state) => state.query
 );
+
+export const totalHits = createSelector(
```

Comment:
seems easier to get this off of results, since if there are results you will definitely have `results.hits.total`, by making another selector that might return null you just have one more null check you have to do